chainservice/chainservice.go

```diff
@@ -138,14 +137,8 @@ func (cs *ChainService) ActionPool() actpool.ActPool {
 	return cs.actpool
 }
 
-// APIServer defines the interface of core service of the server
-type APIServer interface {
-	GetActions(ctx context.Context, in *iotexapi.GetActionsRequest) (*iotexapi.GetActionsResponse, error)
-	GetReceiptByAction(ctx context.Context, in *iotexapi.GetReceiptByActionRequest) (*iotexapi.GetReceiptByActionResponse, error)
-}
-
-// APIServer returns the API server
-func (cs *ChainService) APIServer() APIServer {
+// APIServer returns the grpc server
+func (cs *ChainService) APIServer() *api.GRPCServer {
 	return cs.api.GrpcServer
```

Comment:
I think `CoreService` Interface should be returned here

Reply:
if return CoreService interface,  the final called function [GetActionByActionHash](https://github.com/iotexproject/iotex-core/blob/c39701e6baa2ad5a06eaaa3c4f307db7699f767a/tools/util/injectorutil.go#L841)  and [GetReceiptByAction](https://github.com/iotexproject/iotex-core/blob/c39701e6baa2ad5a06eaaa3c4f307db7699f767a/tools/util/injectorutil.go#L854) will convert CoreService interface to GrpcService.  Can this be better ?

Reply:
Done. Thanks.